markdown/org/docs/patterns/lunetius/instructions/en.md

```diff
@@ -1,14 +1,23 @@
-<Note>
+---
+title: Sewing
```

Comment:
Please use `Instructions` as title because that's what we use everywhere.

Reply:
Done!